_DeploymentAndDistroPackaging/ansible/roles/docker/meta/main.yml

```diff
@@ -27,7 +27,9 @@ galaxy_info:
   platforms:
   - name: Ubuntu
     versions:
-      - xenial
+      - Trusty
```

Comment:
Do these capitalisations work? Have you tested this on all three variants listed?